### DISCUSSION TITLE
How to remove legend from a graph?

### DISCUSSION THREAD

**Discussion by egil** [ipax77/pax.BlazorChartJs#19] [Q&A]
I have been looking through the options types, but have yet to find a way to remove the legend display from graphs (I think it is called the legend, it's the "# of votes" and the red square in the screenshot below). 

![image](https://user-images.githubusercontent.com/105649/219164793-337dc81f-7d46-4e76-978e-0d725493f922.png)

How to remove the legend display from a graph?

**Comment by ipax77** [comment]
Options -> Plugins -> Legend
e.g. 
``` csharp
Options = new ChartJsOptions()
{
    Plugins = new Plugins()
    {
        Legend = new Legend()
        {
            Display = false,
            Labels = new Labels()
            {
                Color = "#CED0DD",
                BoxHeight = 0,
                BoxWidth = 0
            }
        }
}
```
